src/main/java/com/perso/sports/controller/MovementController.java

```diff
@@ -0,0 +1,36 @@
+package com.perso.sports.controller;
+
+import com.perso.sports.entity.MovementEntity;
+import com.perso.sports.entity.RequestAddMovementPresenter;
+import com.perso.sports.repository.MovementRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+public class MovementController {
+    @Autowired
+    MovementRepository movementsRepository;
+
+    @GetMapping("/movement")
```

Comment:
Comme vue ensemble on peut mettre un "s" dans l'url.
```suggestion
    @GetMapping("/movements")
```